python/lsst/ctrl/mpexec/cli/script/report.py

```diff
@@ -24,34 +24,86 @@
 #
 # You should have received a copy of the GNU General Public License
 # along with this program.  If not, see <http://www.gnu.org/licenses/>.
+import pprint
 
+import yaml
+from astropy.table import Table
 from lsst.daf.butler import Butler
 from lsst.pipe.base import QuantumGraph
 from lsst.pipe.base.execution_reports import QuantumGraphExecutionReport
 
 
-def report(butler_config: str, qgraph_uri: str, output_yaml: str, logs: bool = True) -> None:
-    """Write a yaml file summarizing the produced and missing expected datasets
-    in a quantum graph.
+def report(
+    butler_config: str,
+    qgraph_uri: str,
+    full_output_filename: str | None,
+    logs: bool = True,
+    show_errors: bool = False,
+) -> None:
+    """Summarize the produced and missing expected dataset in a quantum graph.
 
     Parameters
     ----------
-    butler_config : `str`
-        The Butler used for this report. This should match the Butler used
-        for the run associated with the executed quantum graph.
-    qgraph_uri : `str`
-        The uri of the location of said quantum graph.
-    output_yaml : `str`
-        The name to be used for the summary yaml file.
-    logs : `bool`
-        Get butler log datasets for extra information.
-
-    See Also
-    --------
-    lsst.pipe.base.QuantumGraphExecutionReport.make_reports : Making reports.
-    lsst.pipe.base.QuantumGraphExecutionReport.write_summary_yaml : Summaries.
+        butler_config : `str`
```

Comment:
The indenting looks a little funny here relative to the old doc string, like it's an extra 4 spaces over.

Reply:
Agreed. I'm kind of surprised that the numpydoc validator doesn't spot this -- maybe I've missed some configuration setting for numpydoc. `velin` (which you have to install from pypi) does notice and fix it.